install.bash

```diff
@@ -278,9 +276,9 @@ else
                 _kaldi_clean ;;
 
             --tue )
-                _kaldi_check_dependencies
-                _kaldi_build
-                _kaldi_online_gst
+                _kaldi_check_dependencies || return 1
```

Comment:
Include print statement so which part fails?

Reply:
Inside each sub routine there is a print statement and return so that the exact issue can be tracked. So I guess a print statement here may not be required